Index: Professional Journal/Amazon Prep.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Professional Journal/Amazon Prep.md b/Professional Journal/Amazon Prep.md
--- a/Professional Journal/Amazon Prep.md	
+++ b/Professional Journal/Amazon Prep.md	
@@ -58,7 +58,6 @@
 -    Domain Driven Design
 -    On system design, we are talking more about different components and how those interact between each other on a high level on a product that we are building
 
-#### Things I'm strong on:
 - Nth Tier Architecture (Front-end, Middle Tier, Back-end)
 - Microservice Oriented Architecture (Payment Service, Fulfilment Service, etc)
 - Data Design (User table, Order Table, Product Table, ORM like Hybernate)
